### DISCUSSION TITLE
Osu crashed my computer

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#16367] [Q&A] [answered]
https://youtube.com/watch?v=leStP7IUeFY
I don't know if it's a bug or something but I found it quite funny

**Comment by 0xdeadbeer** [comment]
Can you tell us more about your computer? Which operating system you're using, etc. 
Also.. can you try posting the osu! logs in here (like runtime.log?) make sure they don't contain any private information tho!

**Reply by ghost** [reply]
Im using arch linux,
[database.log](https://github.com/ppy/osu/files/7833559/database.log)
[network.log](https://github.com/ppy/osu/files/7833560/network.log)
[performance.log](https://github.com/ppy/osu/files/7833561/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7833562/runtime.log)

**Comment by peppy** [comment, marked as answer]
This is an issue with your drivers or hardware. I'd look at making sure they are up-to-date as a first measure. Not much we can do unfortunately.

**Reply by bdach** [reply]
going off of the logs above i'd also recommend trying turning off hardware video decoding in settings, there's a fair few errors related to that there.
